lib/phoenix/endpoint/cowboy_handler.ex

```diff
@@ -104,9 +122,5 @@ defmodule Phoenix.Endpoint.CowboyHandler do
     end
   end
 
-  defp info(scheme, endpoint, config) do
```

Comment:
To repeat what I said on IRC, the same phoenix application can run on 10 different servers on 10 different ports and the endpoint.url would be the same, "example.com:80" which would be proxied to those servers. So we should change the code below to print something like this:

    Running #{inspect endpoint} with Cowboy using #{scheme} on port #{port}